### ISSUE TITLE
remove NFL files no longer being actively used

### ISSUE THREAD

**Issue by raj-al-ghul** [OneiricArts/CustomizeNewTab#18]
transition files:
 
- old NFL file: `NFL.js` (re-written as `NFL_new.js` ... I know I know, its a mess, but thats why I'm fixing this)

- `NFLoff.js` & `NFLpost.js` (back when I did not know about the post-season API until it was too late and had to bootstrap a new version for the post and offseason, now I can design one class to handle all cases)

files for features that I tried out:

- `NFLnews.js` & `NFLdates.handlebars` & `rNFL.handlebars` (used for displaying r/NFL feed in offseason. no longer using, but can make a universal class for showing subreddits and use that for days without games. also used for displaying a countdown to important dates, for which I can also make a universal solution)

these should not definitely not be in `master` in their current form. but good ideas, I can re-incorporate them after more universal re-implementations. start a new branch?

in conclusion, remove: `NFL.js`, `NFLoff.js`, `NFLpost.js`, `NFLnews.js`, `NFLdates.handlebars`, `rNFL.handlebars`

**Comment by raj-al-ghul** [comment]
now I can rename `NFL_new.js` -> `NFL.js`
